collections/_article/dev-snapshot-godot-4-3-dev-4.md

```diff
@@ -0,0 +1,140 @@
+image_caption_title: "Parking Garage Rally Circuit"
+image_caption_description: "A game by Walaber"
+date: 2024-02-29 18:00:00
+---
+
+4.3 dev 4 is jam-packed with changes after 3 weeks of further development. This is an unusually large dev release containing many bug fixes, stability improvements, and new features.
+
+This release includes many notable changes including:
+
+- AnimationMixer feature parity and other animation goodies.
+- .NET code quality and usability improvements.
+- Fixes for invalid/corrupt scenes.
+- UFBX as a built-in alternative to FBX2glTF.
+- Huge improvement to pixel stability for pixel art games.
+- Many rendering features and enhancements.
+- Redesigned graph editor for Visual Shaders.
+- And more!
+
+Keep in mind that while we try to make sure each dev snapshot is stable enough for general testing, this is by definition a pre-release piece of software. Be sure to make frequent backups, or use a version control system such as Git, to preserve your projects in a case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.3.dev4/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+-----
+
+*The illustration picture is from* [**Parking Garage Rally Circuit**](https://store.steampowered.com/app/2737300/Parking_Garage_Rally_Circuit/), *a delightfully retro arcade rally racing that looks straight out of 1998! It is developed by [Walaber](https://twitter.com/walaber) of JellyCar fame, and this upcoming opus is developed using Godot 4.2. Wishlist the game on [Steam](https://store.steampowered.com/app/2737300/Parking_Garage_Rally_Circuit/), and follow the developer on [Twitter](https://twitter.com/walaber) and [Discord](https://discord.com/invite/Ws2MTXyh).*
+
+## Highlights
+
+This snapshot comes loaded with new features and important fixes, after three weeks of further development.
+
+As a reminder, this section only covers changes made since the previous [4.3 dev 3 snapshot](/article/dev-snapshot-godot-4-3-dev-3/). For a more comprehensive overview of what's new in Godot 4.3 compared to 4.2, you'll have to wait for the first beta release, or refer to our [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.3).
+
+### AnimationMixer feature parity and other animation goodies
+
+AnimationMixer continues to receive several fixes and enhancements after being introduced in 4.2 to bring it up to parity with AnimationPlayer.
+
+- [GH-86629](https://github.com/godotengine/godot/pull/86629) adds a `CallbackModeDiscrete` option to AnimationMixer to significantly improve the behaviour when blending continuous and discrete tracks.
```

Comment:
```suggestion
- [GH-86629](https://github.com/godotengine/godot/pull/86629) adds a `CallbackModeDiscrete` option to AnimationMixer to significantly improve the behavior when blending continuous and discrete tracks.
```

Reply:
I can't keep my Canadian voice out of my writing it seems